src/Microsoft.VisualStudio.ProjectSystem.Managed/PooledObjects/PooledStringBuilder.cs

```diff
@@ -52,6 +52,15 @@ public string ToStringAndFree(int startIndex, int length)
             return result;
         }
 
+        public string InternStringAndFree()
+        {
+            var st = StringTable.GetInstance();
```

Comment:
We grab a string table and never release it?

Reply:
Agree that this looks incorrect.